src/qt/bitcoin.cpp

```diff
@@ -251,6 +251,10 @@ BitcoinCore::BitcoinCore(interfaces::Node& node) :
 void BitcoinCore::handleRunawayException(const std::exception *e)
 {
     PrintExceptionContinue(e, "Runaway exception");
+    try {
+        m_node.appShutdown();
+    } catch (...) {
+    }
```

Comment:
Could log something? Same below.

Reply:
Any suggestions? Note that I'd rather not write to the debug.log, since that might be the cause of the exception.

Reply:
I tend to agree that blanket-ignoring exceptions is very bad. Even printing to stderr would be better to at least have a potential chance of someone diagnosing of seeing the error!

Reply:
The gui is not always started from the command line, so that would be missed often as well. But I guess I could add it.